anjani/util/tg.py

```diff
@@ -102,8 +102,10 @@ def revert_button(button: Button) -> str:
     return res
 
 
-def parse_button(text: str) -> Tuple[str, Button]:
-    """Parse button to save"""
+def parse_button(text: str) -> Tuple[str, List[Tuple[str, str, bool]]]:
+    if text is None:
```

Comment:
should be handled on `get_message_info()`
`parse_button(text: str)` only expect input string so there should no  `None`  check here